docs/tutorials/1.5/index.md

```diff
@@ -0,0 +1,12 @@
+# Chrysalis IOTA 1.5
+
+Chrysalis is currently in a test stage. The release is planned for Q1 2021.
```

Comment:
We're already out of Q1

Reply:
This page is left behind - and will be probably removed at the end of the day.